addons/base_import_module/models/ir_module.py

```diff
@@ -52,7 +63,7 @@ def _import_module(self, module, path, force=False):
                     continue
                 _logger.info("module %s: loading %s", module, filename)
                 noupdate = False
-                if filename.endswith('.csv') and kind in ('init', 'init_xml'):
+                if ext == '.csv' and kind in ('init', 'init_xml'):
```

Comment:
I changed this part although it doesn't have anything to do with the task at hand because it looked silly, ext is already defined so no need to calculate .endswith()